.travis.yml

```diff
@@ -9,6 +9,13 @@ sudo: required
 env:
   global:
     - ANSIBLE_CONFIG=tests/ansible.cfg
+  matrix:
+    - RUN="shellcheck yamlcheck syntax ci" SITE="global_vars/default-site.yml"
+    - RUN="ci" SITE="tests/site_vars/l2tp.yml"
+    - RUN="ci" SITE="tests/site_vars/openvpn.yml"
+    - RUN="ci" SITE="tests/site_vars/shadowsocks.yml"
+    - RUN="ci" SITE="tests/site_vars/ssh.yml"
+    - RUN="ci" SITE="random"
```

Comment:
Another entry for OpenConnect?

Reply:
I was initially thinking that a build task for each service in isolation was too many and so I'd pick a few services to do that with and then let the random build catch the rest.

Since it ended up being _only_ OpenConnect that isn't represented here I might as well add it :-)